src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -41,6 +45,29 @@
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+
+        final DateTimeFormatter Formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
https://stackoverflow.com/questions/66066/what-is-the-best-way-to-implement-constants-in-java

Reply:
Thanks! Done.